src/contexts/Graph.js

```diff
@@ -160,10 +186,20 @@ const Graph = ({ children }) => {
       event.txnLink = `https://etherscan.io/tx/${event.txnHash}`
       eventsArray.push(event)
     })
-    graphRinkebyData.data.newReportEntities.forEach((event) => {
+    /*graphRinkebyData.data.newReportEntities.forEach((event) => {
```

Comment:
Rinkeby stuff is commented out. Can be removed? What about the other Rinkeby constants & functions? Can also be removed?

Reply:
I only commented out this part so it didn't show up on the dropdown list. Wasn't ready to completely purge for fear of breaking something.